### ISSUE TITLE
Improperly configured wiki permissions

### ISSUE THREAD

**Issue by s-i-l-k-e** [koalaman/shellcheck#1864]
I'm able (as a user with no association at all with the project) to create new pages as well as edit all wiki pages. To prevent abuse, wi
Example: https://github.com/koalaman/shellcheck/wiki/SC1027
Apologies for creating the page, I genuinely didn't expect it to work.

**Comment by gromgit** [comment]
Good point. Given the maturity of Shellcheck, perhaps it's time to properly publish the master docs under version control (at least the usage and SC pages), and leave the wiki for installation/integration guides and the like.

Perhaps something akin to https://docs.brew.sh/, whose source is part of the Homebrew source tree: https://github.com/Homebrew/brew/tree/master/docs
